src/main/java/de/hhu/stups/plues/prob/MockSolver.java

```diff
@@ -120,8 +124,28 @@ public void move(final String sessionId, final String day, final String slot) {
     return Collections.emptyList();
   }
 
+  @SuppressFBWarnings("DMI_HARDCODED_ABSOLUTE_FILENAME")
```

Comment:
Fix this FindBugs warning by removing the code below and restoring the previous "mock" behaviour